pom.xml

```diff
@@ -102,7 +103,8 @@
     <version.org.simpleframework>6.0.1</version.org.simpleframework>
     <!-- this version will overwrite jboss-ip-bom version 2.5.1. The version in jboss-ip-bom couldn't be upgraded as it will break ModeShape Web Explorer application which uses GWT 2.5.1 -->
     <!-- please look at https://github.com/fredsa/gwt-dnd/wiki/GettingStarted -->
-    <version.com.google.gwt>2.7.0</version.com.google.gwt>
+    <version.com.google.gwt>2.8.0-beta1</version.com.google.gwt>
+    <version.com.google.guava>20.0-SNAPSHOT</version.com.google.guava>
```

Comment:
Seems like we are really using bleeding edge stuff :) I guess we won't be doing any releases from master for some time, so should be acceptable for now.